WordPress/gutenberg
# Use format-library in gutenberg-mobile

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/221

This PR reuses the existing logic for the format buttons in `@wordpress/format-library` instead of our custom solution added in Aztec.

Moreover, it adds a React Native custom modal component to the format-library `Link` component which handles creating and editing links. It also create a RN version of `URLInput`.

This is what it looks like (Not as nice as the mockup yet https://github.com/wordpress-mobile/gutenberg-mobile/issues/221#issuecomment-436409456 but I think we can iterate on it):
![image](https://user-images.githubusercontent.com/230230/50102669-efb25c80-0225-11e9-9223-fd9602f53568.png)


## To test

Check out the gutenberg-mobile PR at https://github.com/wordpress-mobile/gutenberg-mobile/pull/275. Try to use the formatting bar on paragraphs and headings.
